pyvrp/cpp/search/LocalSearch.h

```diff
@@ -56,6 +56,8 @@ class LocalSearch
     // numLocations, but nothing is stored for the depots!)
     Neighbours neighbours_;
 
+    size_t numPerturbations_ = 0;  // number of perturbations to apply
```

Comment:
I still don't really understand what this means or does. Reading it, I think it means the number of times we call the perturbation operators. But that's not it, it's more of a "degree of destruction" kind of thing.